src/pages/About/index.tsx

```diff
@@ -121,12 +122,22 @@ const PoweredBySection = styled(Panels)`
 export default function About() {
   const isDarkMode = useIsDarkMode()
 
+  const titleRef = useRef<HTMLDivElement>(null)
+  const [titleHeight, setTitleHeight] = useState(0)
+  useLayoutEffect(() => {
+    if (titleRef.current) {
+      setTitleHeight(titleRef.current.clientHeight)
```

Comment:
might consider scrollHeight here instead? not sure

Reply:
i can't set scroll height unless there is enough padding to be able to scroll

Reply:
i think that would accomplish the opposite effect. it would scroll further down the page